openwisp_radius/admin.py

```diff
@@ -633,7 +634,10 @@ def has_delete_permission(self, request, obj=None):
 
     Token = apps.get_model('authtoken', 'Token')
 
-    admin.site.unregister(Token)
+    try:
+        admin.site.unregister(Token)
+    except NotRegistered:
+        pass
```

Comment:
Handled this as done [here](https://github.com/openwisp/openwisp-users/blob/a8608381b6c32419b96fb0a48981b5ec4536821e/openwisp_users/admin.py#L674)

It is causing coverage to decrease